#Expyriment 1.0.0 (Python 3.11.3), .xpe-file, coding: cp1252
#date: Wed Sep 17 2025 19:17:28
#sha1: 0aa24c
#modules: 
#display: size=(800, 600), window_mode=True, opengl=2
#os: uname_result(system='Windows', node='RowanLunch', release='10', version='10.0.26120', machine='AMD64')
Time,Type,Event,Value,Detail,Detail2
2567,design,log,
#design: #exp: Circle
#design: block_cnt,block_id,trial_cnt,trial_id
2567,design,logged,
2567,Experiment,started
2568,stderr,received,'Traceback (most recent call last):'
2568,stderr,received,'  File "C:\\Users\\Rowlu\\Documents\\Rowan\\University\\M1 2025-2026\\Courses\\Coding\\assignments\\week-2\\exercises\\two_squares.py", line 21, in <module>'
2568,stderr,received,'    square.present(clear=True, update=False)'
2569,stderr,received,'    ^^^^^^'
2569,stderr,received,"NameError: name 'square' is not defined"
